src/workflow/restore_from_mnemonic.c

```diff
@@ -143,11 +143,11 @@ bool workflow_restore_from_mnemonic(const RestoreFromMnemonicRequest* request)
     UTIL_CLEANUP_32(seed);
     size_t seed_len = 0;
     if (!keystore_bip39_mnemonic_to_seed(mnemonic, seed, &seed_len)) {
-        workflow_status_blocking("Invalid mnemonic", false);
+        workflow_status_blocking("Recovery words\ninvalid", false);
```

Comment:
I guess it doesn't fit on one line?

Reply:
Or how about "Invalid recovery words" to emphasize **invalid**?

Reply:
yeah, doesn't fit on one line.

the status screen also has a big cross, so it is very emphasized :)